### ISSUE TITLE
iOS actionable pushes. Buttons in ios pushes

### ISSUE THREAD

**Issue by locskot** [mwaylabs/flutter-apns#62]

**Comment by locskot** [comment]
> Hi
> I saw this issue #24.
> 
> I've tried to implement actionable pushes but I couldn't.
> 
> 1. Here is what I send:
>    ![image](https://user-images.githubusercontent.com/16982464/122927195-4f4acf00-d371-11eb-9985-dade45149322.png)
> 2. Here is what I receive on iPhone
>    ![image](https://user-images.githubusercontent.com/16982464/122927628-bc5e6480-d371-11eb-8797-6c807c9f7f44.png)
>    ![IMG_EB64CA754E68-1](https://user-images.githubusercontent.com/16982464/122927524-a2bd1d00-d371-11eb-9255-d11bcc12e028.jpeg)
> 3. Here is how I'm trying to manage actionable notifications
>    ![image](https://user-images.githubusercontent.com/16982464/122933080-faaa5280-d376-11eb-95e6-ca787058c320.png)
> 4. Do I need to add something to AppDelegate.swift file? I've tried something (it's commented in the screenshot)
>    ![image](https://user-images.githubusercontent.com/16982464/122932790-b3bc5d00-d376-11eb-8097-be0af456084a.png)

**Comment by locskot** [comment]
Hi
I've tried to implement actionable notifications. I couldn't through. 

1. I'm sending this
{
   "aps":{
      "badge":1,
      "alert":{
         "title":"Awesome title",
         "subtitle":"Awesome subtitle",
         "body":"Awesome body."
      },
      "sound":"default",
      "category":"SOME_CATEGORY"
   }
}

2.  I use this code to manage the notifications
![image](https://user-images.githubusercontent.com/16982464/122939391-3eec2180-d37c-11eb-8641-a4c4c1352ba0.png)


3. Do I need to add something to AppDelegate.swift file. I tried both variants (commented and not commented)
![image](https://user-images.githubusercontent.com/16982464/122939565-693ddf00-d37c-11eb-9974-8ebc30e79af4.png)
